src/main/java/org/radarbase/appserver/exception/FileStorageException.java

```diff
@@ -0,0 +1,13 @@
+package org.radarbase.appserver.exception;
+
+public class FileStorageException extends RuntimeException {
```

Comment:
Can you map these to HTTP codes using `@ResponseStatus(HttpStatus.XYZ)`? Perhaps http code 500 in this case.

Reply:
Added! Can you check this please. I have never used this mechanism.

Reply:
yes looks good, thanks